src/status_im2/contexts/wallet/common/account_options/view.cljs

```diff
@@ -0,0 +1,35 @@
+(ns status-im2.contexts.wallet.common.account-options.view
```

Comment:
It's good to organize the views that are used as sheets in a separate sub-directory as this `common` directory contains views that are used as components too. 

Maybe `status-im2.contexts.wallet.common.sheets.account-options.view`. WDYT?